broadinstitute/sam
# Canshare

https://broadinstitute.atlassian.net/browse/GAWB-2922

enable sharing of policies if a user has the action can_share_{policy name}, the format for specifying the acceptable pattern for action names in config is with a regex

tests:
- add/delete user to/from policy is successful when user has can share for that policy
- add/delete user to/from policy gives when user does not have can share for that policy (existing test slightly tweaked)
- actions may be added to a policy if they match configured regex but not otherwise

.

  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
